i6_models/parts/conformer/mhsa.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Optional, Callable
+import torch
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class ConformerMHSAV1Config(ModelConfiguration):
+    input_dim: int  # input dim and total dimension for query/key and value projections, should be dividable by `num_att_heads`
+    num_att_heads: int  # number of attention heads
+    key_padding_mask: Optional[
+        torch.Tensor
+    ] = None  # binary or float mask deciding if a key position should ignored or not, of shape (B, T_k)
+    att_weights_dropout: float = 0.1  # attention weights dropout
```

Comment:
Have we decided whether to add defaults for some values like dropout yet? I know there was a discussion, but I didn't follow the results. @JackTemaki @albertz ?